content/docs/04-clusters/01-public-cloud/05.5-cox-edge/90-network-rules.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Network Rules"
```

Comment:
```suggestion
title: "Required Network Rules"
```

Reply:
I'd include 'Required' in the title to emphasize requirements, and it's consistent with **AWS > Required IAM Policies**

Reply:
Good point 😃